pages/templates/pages/base_page.html

```diff
@@ -91,10 +95,12 @@ <h1>{{ page.title }}</h1>
 
 </div>
 
-{% include "partials/footer.html" %}
-{% include "partials/footer-js.html" %}
+{% block footer %}
+  {% include "partials/footer.html" %}
+  {% include "partials/footer-js.html" %}
 
-{% block footer_scripts %}
+  {% block footer_scripts %}
```

Comment:
Is nested block pattern correct here? Maybe should be separate blocks? Not sure what footer_scripts is used for.

Reply:
This should probably all just go in a footer partial. Footer scripts are javascript that need to go in footer like analytics stuff I believe.

Reply:
Ok so that was exactly what I was thinking about. We still want analytics even if not showing the main footer right?

Reply:
Gotcha, yes.

Reply:
I think this is potentially critical - we want analytics on this page right? Do we need to fix before deploy today?